### PR TITLE
Add @minutely as pre-defined schedule

### PR DESCRIPTION
**Pull request by mlafeldt**
In addition to @hourly, @daily, etc.

### CODE REVIEW

File: cronexpr_test.go [hunk 1]
```diff
@@ -278,7 +278,7 @@ func TestNextN_every5min(t *testing.T) {
 	for i, next := range result {
 		nextStr := next.Format("Mon, 2 Jan 2006 15:04:05")
 		if nextStr != expected[i] {
-		t.Errorf(`MustParse("*/5 * * * *").NextN("2013-09-02 08:44:30", 5):\n"`)
+			t.Errorf(`MustParse("*/5 * * * *").NextN("2013-09-02 08:44:30", 5):\n"`)
```
Comment (mlafeldt) [inline cronexpr_test.go]: Note: This was automatically formatted by gofmt.

**Comment by mlafeldt** [comment]
@gorhill PTAL

**Comment by gorhill** [comment]
I don't see minutely defined anywhere as a valid cron expression, I rather to stick to what is documented elsewhere. Add to this that the [English definition for _minutely_](http://www.oxforddictionaries.com/definition/english/minutely) is:

> With great attention to detail; meticulously

**Comment by mlafeldt** [comment]
Alright.

**Pull request closed without merging**